exec.go

```diff
@@ -82,13 +82,13 @@ func isexecc(c rune) bool {
 }
 
 func printarg(argt *Text, q0 int, q1 int) string {
-	if argt.what != Body || argt.file.name == "" {
+	if argt.what != Body || argt.oeb.f.details.Name == "" {
```

Comment:
Write a `Name()` wrapper method inside of `OldEditableBuffer`. You'll want to have done this anyway for the subsequent CL where you move `details` out of `File` and into `OldEditableBuffer`.